### PR TITLE
create auth via facebook google

### PR DESCRIPTION
**Pull request by lamngockhuong**
https://dev.framgia.com/redmine/issues/156410

### CODE REVIEW

**Comment by lamngockhuong** [comment]
ready

File: resources/views/public/404.blade.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+@section('page-heading-breadcrumbs-section')
+    <h2>@lang('public.404')</h2>
+    <ul class="breadcrumbs">
+        <li><a href="{{ route('home') }}">@lang('public.home')</a></li>
+        <li>@lang('public.404')</li>
+    </ul>
+@endsection
+@section('page-heading-banner')
+    <div class="overlay-dark theme-padding parallax-window" data-appear-top-offset="600" data-parallax="scroll"
+         data-image-src="{{ asset('templates/public/images/inner-banner/img-03.jpg') }}">
```
Comment (at-diephq) [inline resources/views/public/404.blade.php]: convention này em, 4 space nhé

**Comment by at-diephq** [comment]
commented

**Comment by lamngockhuong** [comment]
ready

**Comment by at-diephq** [comment]
(merged)

**Pull request merged**